source/configuration/modules/mmkubernetes.rst

```diff
@@ -0,0 +1,357 @@
+*****************************************
+Kubernetes Metadata Module (mmkubernetes)
+*****************************************
+
+===========================  ===========================================================================
+**Module Name:**             **mmkubernetes**
+**Author:**                  `Tomáš Heinrich`
+                             `Peter Portante` <pportant@redhat.com>
```

Comment:
You can remove my name here.  While I moved and submitted the first PR, I don't want to claim authorship and misrepresent that.